test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -451,5 +477,92 @@ private static void DeleteDirectory(string path)
 
             }
         }
+
+        //temporary added methods for processing deps.json files for patching
+        private void TempPatching(string patchDir)
```

Comment:
Read [.NET type member naming conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members) and rename `TempPatching`, `TempCopyNewlyAddedDlls`, and `TempPatchingDepsJsonForNewlyAddedDlls`.